server/hedley/modules/custom/hedley_admin/scripts/generate-demographics-report.php

```diff
@@ -351,8 +351,8 @@ function group_encounter_unique($measurement_types_list, $limit = NULL) {
   ],
   [
     '   CHW',
-    encounter_all_count('prenatal', 'all', $limit_date) - encounter_all_count('prenatal', 'chw', $limit_date),
-    encounter_unique_count('prenatal', 'all', $limit_date) - encounter_unique_count('prenatal', 'chw', $limit_date),
+    encounter_all_count('prenatal', 'all', $limit_date) - encounter_all_count('prenatal', 'hc', $limit_date),
```

Comment:
@adamhstewart Are these intentional fixes / changes?

Reply:
I think those are from another PR that is already merged.